.gitpod.dockerfile

```diff
@@ -2,8 +2,11 @@ FROM gitpod/workspace-postgres
 
 # Install Ruby
 ENV RUBY_VERSION=3.0.2
-RUN rm /home/gitpod/.rvmrc && touch /home/gitpod/.rvmrc && echo "rvm_gems_path=/home/gitpod/.rvm" > /home/gitpod/.rvmrc
+
+# Taken from https://www.gitpod.io/docs/languages/ruby
+RUN echo "rvm_gems_path=/home/gitpod/.rvm" > ~/.rvmrc
```

Comment:
From the reference link in the comment.

> Explanation: Gitpod initially sets up RVM in /home/gitpod/.rvm, but then later switches the RVM configuration directory to /workspace/.rvm, so that any user-made changes (like installing new gems) are persisted within a Gitpod workspace. However, during the Dockerfile build, the /workspace directory doesn’t exist yet, so we temporarily reset RVM’s configuration directory to /home/gitpod/.rvm.